Bot_project/core.cpp

```diff
@@ -0,0 +1,71 @@
+#include <iostream>
+#include <string>
+
+using namespace std;
+
+bool prompt(const string& user_name )
+{
+    //declaring the variables locally.
+    string user_input, answer;
+    string greetings[] = {"hello", "hi", "greetings", "how are you", "wussup"};
+
+
+    cout << "You: ";
+    getline(cin, user_input); // using *getline so it takes in strings with spaces.
+
+    //a for loop to turn Input into lowercase
+    for(char &user_input: user_input)
+        user_input = tolower(user_input); //finally used std::tolower :)
+
+    //if statement to answer only to specific strings
+    if (user_input == greetings[0] || user_input == greetings[1])
```

Comment:
to make the bot a bit smarter, does the greeting have to be an exact match?! How can you make it more flexible? For example, if the user types 'hello there', does it need a new entry in the greetings array or can you modify the how we're checking?